development-docs/TESTING.md

```diff
@@ -276,8 +276,6 @@ The following table shows currently used tags:
 | mirrormaker2       |                 Execute tests that deploy KafkaMirrorMaker2 resource                  |
 | conneccomponents   |  Execute tests that deploy KafkaConnect, KafkaMirrorMaker2, KafkaConnector resources  |
 | bridge             |                          Execute tests that use Kafka Bridge                          |
-| internalclients    |         Execute tests that use internal (from the Pod) Kafka clients in tests         |
-| externalclients    |          Execute tests that use external (from code) Kafka clients in tests           |
```

Comment:
I would personally keep the `externalclients` as I'm using it sometimes when I'm running the tests locally or on some different env when you want to try tests where we are sending/receiving messages externally (IIRC not every Nodeport or Loadbalancer related ST has external clients there).

Reply:
sounds reasonable 👍 Will keep external clients tag ... thanks for giving head up about usage!